hooks/boot/01-helpers.rb

```diff
@@ -7,7 +7,8 @@ def module_enabled?(kafo, name)
     end
 
     def log_and_say(level, message)
-      say "<%= color('#{message}', :#{level.to_s}) %>"
+      style = level == :error ? 'bad' : level
```

Comment:
Logger + colorization match for "info" but for whatever reason the color of error is "bad".